agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -803,7 +803,7 @@ bool main_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
                                       std::chrono::seconds(PLATFORM_READ_CONF_RETRY_SEC);
                     }
                 } else {
-                    usleep(PLATFORM_READ_CONF_SLEEP_USEC);
+                    sleep(PLATFORM_READ_CONF_RETRY_SEC);
```

Comment:
Dude... You're still not using the `retry_timer` variable.
Anyway, this entire loop can be simplified:
```
register_response->valid() = 0;
do {
	LOG(TRACE) << "Trying to read settings of iface:" << strIfaceName
			   << ", attempt=" << int(retry_cnt);

	if (fill_platform_settings(strIfaceName, register_response,
							   platform_common_conf, bpl_iface_wlan_params_map,
							   sd)) {
		register_response->valid() = 1;
	} else {
		LOG(INFO) << "Reading settings of iface:" << strIfaceName
				  << ", attempt=" << int(retry_cnt) << " has failed!";
		
		// Reached the maximal number of retries, stop trying...
		if (++retry_cnt == PLATFORM_READ_CONF_MAX_ATTEMPTS)
			break;

		// Sleep and retry
		std::this_thread::sleep_for(std::chrono::seconds(PLATFORM_READ_CONF_RETRY_SEC));
	}
	
} while (!register_response->valid() && !should_stop);
```

Reply:
ok - refactored the loop to what you suggested